source/index.html.md

```diff
@@ -1301,12 +1302,63 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
     <div class="type">May not be all digits, all same character, or all sequential characters (e.g. "abc")</div>
     <div class="type">Optional</div>
   </dd>
+  <dt>travel
+    <span class="type">dictionary</span>
+  </dt>
+  <dd>
+    See <a href="#travel-data">Travel data</a>.
+    <div class="type">Optional</div>
+  </dd>
 </dl>
 
 <p class="alert alert-info">
-<b>Notice:</b> A capture cannot be made if the authorization is 180 days old.
+<b>Notice:</b> A capture cannot be made if the authorization is 180 days old or more.
 </p>
 
+
+##### Travel data
+
+At most one type of travel data can be supplied for a capture; if `travel` is
+supplied, it must include exactly one of `travel[car][...]`,
+`travel[flight][...]`, or `travel[lodging][...]`.
```

Comment:
```suggestion
supplied, it must include exactly one of `travel[car]`, `travel[flight]`, or
`travel[lodging]`.
```

I think `[...]` is superfluous here.

Reply:
I think I introduced these, and I'm happy to remove them, so I'll just commit the suggestion 🙂